src/app/static/src/app/components/plots/choropleth/Choropleth.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+    <div>
+        <l-map ref="map" style="height: 800px; width: 100%" @ready="updateBounds">
+            <l-geo-json v-for="feature in currentFeatures"
+                        ref="featureRefs"
+                        :key="feature.properties.area_id"
+                        :geojson="feature"
+                        :options="createTooltips"
+                        :options-style="() => {return {...style, fillColor: getColour(feature)}}">
+            </l-geo-json>
+            <reset-map @reset-view="updateBounds"></reset-map>
+            <map-legend :indicator-metadata="indicatorMetadata"
+                        :scale-levels="scaleLevels"
+                        :selected-scale="selectedScale"
+                        @update:selected-scale="updateColourScales"></map-legend>
+        </l-map>
+    </div>
+
+</template>
+
+<script lang="ts">
+import {computed, defineComponent, onMounted, ref, watch} from "vue";
+import {useStore} from "vuex";
+import {RootState} from "../../../root";
+import {PlotData} from "../../../store/plotData/plotData";
+import { LMap, LGeoJson } from "@vue-leaflet/vue-leaflet";
+import { Feature } from "geojson";
+import {
+    getVisibleFeatures,
+    getFeatureData,
+    initialiseScaleFromMetadata,
+} from "./utils";
+import ResetMap from "./ResetMap.vue";
+import MapLegend from "./MapLegend.vue";
+import {getColourRange, getScaleLevels} from "../utils";
+import {IndicatorValuesDict, NumericRange} from "../../../types";
+import {ChoroplethIndicatorMetadata} from "../../../generated";
+import { ScaleSettings } from "../../../store/plotState/plotState";
+import {useMapTooltips} from "../useMapTooltips";
+import {useUpdateScale} from "../useUpdateScale";
+
+export default defineComponent({
+    setup() {
+        const store = useStore<RootState>();
+        const plotData = computed<PlotData>(() => store.state.plotData.choropleth);
+
+        const {updateOutputColourScale} = useUpdateScale();
```

Comment:
Are we not going to use the same choropleth for input data too? Do we not need to tell the choropleth which data and updates to use, or will that come in a future ticket?

Reply:
Yes we will need to, just wanted to leave that for a later ticket